### PR TITLE
Navbar fix

### PR DESCRIPTION
**Pull request by SafeEHA**
Did a few fixes on the navbar 

- Made navbar static
- Moved the user guide button to the navbar and made it a secondary button
- Centered nav links
- Added the search logo, making search more visible

The first PR #317 was closed due to some clarity issues, I hope you'll find this easier to review 
cc: @elfosardo @hroyrh

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -25,24 +25,43 @@
           </svg>
           <span class="mk-main-header__toggle-text">menu</span>
         </button>
+      </nav>
```
Comment (hroyrh) [inline _includes/header.html:28]: Any reason for closing navigation earlier ?

Comment (SafeEHA) [reply _includes/header.html:28]: While trying to center the nav links on mobile responsiveness, I couldn't get them to center perfectly so I decided to separate the toggle button and create a separate div (ul) making the nav links stand independently.
Thanks for raising this, I would change the nav to div for best practice

File: _sass/_global.scss [hunk 2]
```diff
@@ -37,8 +37,8 @@
             width: 15px;
             height: 15px;
             display: inline-block;
-            position:relative;
-            left: var(--mk--spacer--sm);
+            // position:relative;
```
Comment (elfosardo) [inline _sass/_global.scss:40]: if this is not used let's remove it

File: _sass/_homepage.scss [hunk 3]
```diff
@@ -1,6 +1,6 @@
 
 .mk-community-callout {
-    position: relative; //shadow fix
+    // position: relative; //shadow fix
```
Comment (elfosardo) [inline _sass/_homepage.scss:3]: ditto

**Review by elfosardo — CHANGES_REQUESTED** [review]

File: _includes/header.html [hunk 4]
```diff
@@ -25,24 +25,43 @@
...
+              <button type="submit" id = "search-button" class = "search-button" disabled = 'true' >
+                <img src="../assets//images/search.png" style="height: 20px;" alt="">
+    
+              </button>
+              <!-- <input id="search-button" class="search-button" type="submit" disabled='true' /> -->
+            </form>
+          </li>
+    
         </ul>
-      </nav>
```
Comment (hroyrh) [inline _includes/header.html:45]: should that div block at line 28 closed here ?

File: _includes/header.html [hunk 5]
```diff
@@ -25,24 +25,43 @@
...
+          <li {% if page.url == "/search.html" %} class="active" {% endif %}>
+            <form action="/search.html" method="get" autocomplete="off" class="mk-search-form">
+              <div class="autocomplete" style="width:150px;">
+                <input type="text" id="search-input" class="docs-search--input" placeholder="Search Term" name="query">
+              </div>
+              <button type="submit" id = "search-button" class = "search-button" disabled = 'true' >
+                <img src="../assets//images/search.png" style="height: 20px;" alt="">
+    
+              </button>
+              <!-- <input id="search-button" class="search-button" type="submit" disabled='true' /> -->
```
Comment (hroyrh) [inline _includes/header.html:60]: remove this if its not needed

File: _includes/header.html (hunk 4 above)
Comment (SafeEHA) [reply _includes/header.html:45]: No, it doesn't close here, It was the nav from yesterday I changed to a div for the best practice

File: _includes/header.html (hunk 5 above)
Comment (SafeEHA) [reply _includes/header.html:60]: Ok sure

**Pull request merged**